certbot/certbot/_internal/storage.py

```diff
@@ -911,7 +911,7 @@ def ocsp_revoked(self, version):
             return ocsp.RevocationChecker().ocsp_revoked_by_paths(cert_path,
                                                                   chain_path)
         except Exception as e:  # pylint: disable=broad-except
-            logger.warning(
+            logger.error(
```

Comment:
Do you think we should display OCSP errors in `--quiet`/cron output? 

My feeling is that `warning` is appropriate because OCSP checking is only a soft failure.

Reply:
You're right, nice catch.